tests/x11/gnomecase/change_password.pm

```diff
@@ -51,7 +51,8 @@ sub reboot_system {
     assert_screen "displaymanager", 200;
     $self->{await_reboot} = 0;
     # The keyboard focus is different between SLE15 and SLE12
-    send_key 'up' if sle_version_at_least('15');
+    send_key 'up'  if is_sle('15+');
+    send_key 'tab' if is_opensuse;
```

Comment:
why I use is_opensuse is because Tumbleweed and Leap all open "auto-login" after installation. Following are all same reason to use is_opensuse.

Reply:
Send ‘tab’ here is because the username select focus lost after reboot on opensuse